google-client-loader.html

```diff
@@ -195,12 +144,27 @@
         }
       },
 
+      /**
+       * Wrapper for `gapi.auth`.
+       * @property auth
+       * @type gapi.auth
+       */
+      get auth() {
```

Comment:
It works, I have no preference. The way it looks in the docs might swing the vote.